client/src/screens/groups/styles.js

```diff
@@ -6,6 +6,65 @@ const FONT_SIZE = 16;
 const HIGHLIGHT_COLOR = 'rgba(0,118,255,0.9)';
 
 const styles = StyleSheet.create({
+  topContainer: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    elevation: 1,
+    borderRadius: 2,
+    backgroundColor: 'white',
+    paddingTop: 10,
+    paddingBottom: 10,
+    paddingLeft: 8,
+    paddingRight: 8,
+    marginLeft: 10,
+    marginRight: 10,
+    marginTop: 6,
+    marginBottom: 0,
+  },
+  leaveGroupText: {
+    fontSize: 7,
+    includeFontPadding: false,
+    flex: 0,
+    color: '#6B6B6B',
```

Comment:
In future - can we move these colors to themes/colors.js

Reply:
deleted them.. so that fixes that 👍